share_app/src/main/res/values/strings.xml

```diff
@@ -68,4 +68,8 @@
     <string name="no_forms_sent">No filled %s forms sent</string>
     <string name="no_forms_received">No filled %s forms received</string>
     <string name="no_forms_reviewed">No finalized %s forms received</string>
+
+    <string name="num_forms_sent">Number of forms sent</string>
```

Comment:
Maybe consider using "filled forms" here? The context makes it pretty clear but I always think of "form" as "blank form".